src/poliastro/plotting/static.py

```diff
@@ -209,15 +209,19 @@ def _plot(self, trajectory, state=None, label=None, color=None):
 
         return lines
 
-    def plot(self, orbit, label=None, color=None, method=mean_motion):
+    def plot(self, orbit, label=None, color=None, **kwargs):
```

Comment:
Don't change the function signature. Make the import inside the function. This will solve the problem of circular imports

Reply:
See one of my next comments